ports/zephyr/modules/_boot.py

```diff
@@ -2,13 +2,13 @@
 import vfs
 import sys
 import os
-from zephyr import FlashArea
+from zephyr import FlashArea, DiskAccess
```

Comment:
You need to check if the modules exist before doing the relevant things with them, as they will be missing if not provided